ios/Podfile.lock

```diff
@@ -1485,7 +1485,7 @@ SPEC CHECKSUMS:
   FBLazyVector: 56e0e498dbb513b96c40bac6284729ba4e62672d
   FBReactNativeSpec: 146c741a3f40361f6bc13a4ba284678cbedb5881
   fmt: ff9d55029c625d3757ed641535fd4a75fedc7ce9
-  glog: c5d68082e772fa1c511173d6b30a9de2c05a69a2
+  glog: 530710e7949eb12c82670bd09e946becf50a3c2a
```

Comment:
I don't use macOS to develop, but is this change really needed?

Reply:
Hey @qfrank : the glog hash change is indeed something that should not happen and whenever we install pods we get a different hash from the place we fetch glog.
I logged a similar issue with jitpack for blur dependency -> https://github.com/jitpack/jitpack.io/issues/6494
We should not gitignore the podfile.lock
> Whether or not you check in the Pods directory, the Podfile and Podfile.lock should always be kept under version control.

ref -> https://guides.cocoapods.org/using/using-cocoapods.html#benefits-of-ignoring-the-pods-directory